io.github.hydrusnetwork.hydrus.yml

```diff
@@ -0,0 +1,127 @@
+app-id: io.github.hydrusnetwork.hydrus
+
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.3'
+
+command: hydrus.sh
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is very broad filesystem permission.

Reply:
I was not sure about giving this permission. This app relies a lot on files, you can monitor directories, download images to disk, store everything on network location etc. I feel like removing this permission would be too harsh for users since it would result in them having to handle specific permission for every situation... What do you think ?

Reply:
what I think is that `host` is way too broad.